pyclaragenomics/claragenomics/bindings/cudapoa.pyx

```diff
@@ -99,23 +108,25 @@ cdef class CudaPoaBatch:
             st = stream.stream
             temp_stream = <_Stream>st
 
-        cdef int8_t output_mask
```

Comment:
I think this change needs to be reverted. the API exposed the output type to be a string - "consensus" or "msa". that's because we don't expose the cdef cudapoa.consensus type publicly